src/AutoRest.CSharp/Common/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -12,46 +12,60 @@
 using AutoRest.CSharp.AutoRest.Plugins;
 using AutoRest.CSharp.Common.Input;
 using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Output.Models.Types;
 using Azure.Core;
+using Microsoft.CodeAnalysis;
 
 namespace AutoRest.CSharp.AutoRest.Communication
 {
     internal class StandaloneGeneratorRunner
     {
-        public static async Task RunAsync(string[] args)
+        public static async Task RunAsync(CommandLineOptions options)
```

Comment:
Is there any script change needed?  @chunyu3

Reply:
Yes, updated.